nightwatch.json

```diff
@@ -1,18 +1,18 @@
 {
-  "src_folders" : ["/spec/features"],
+  "src_folders" : ["./spec/features"],
   "output_folder" : "reports",
   "custom_commands_path" : "",
   "custom_assertions_path" : "",
   "page_objects_path" : "",
   "globals_path" : "",
 
   "selenium" : {
-    "start_process" : true,
-    "server_path" : "./node_modules/selenium-standalone/.selenium/selenium-server-standalone-3.9.1.jar",
+    "start_process" : false,
+    "server_path" : "",
     "log_path" : "",
-    "port" : 8080,
+    "port" : 4444,
```

Comment:
Was this changed due to a port conflict? Do we still need to stub websocket in our unit tests?